cmd/bridge/main.go

```diff
@@ -108,6 +108,9 @@ func main() {
 
 	fLoadTestFactor := fs.Int("load-test-factor", 0, "DEV ONLY. The factor used to multiply k8s API list responses for load testing purposes.")
 
+	fHelmChartRepoURL := fs.String("helm-chart-repo-url", "https://redhat-developer.github.com/redhat-helm-charts", "Helm chart repository URL")
```

Comment:
My only remaining question is whether we should default this here or in the operator. As written, there's no way to disable the endpoint (for disconnected clusters that haven't been configured or for cluster admins who just don't want helm)

Reply:
Due to the tight timeline, we would like to default here to our own repo for now, and once this gets merged, we would make it configurable through the operator. Just trying to minimize the number of PRs that need to be merged in order to expose our own chart repo. @sbose78 , wdyt?

Reply:
Here's what the sequence of PRs should look like:

1. In this PR, default "helm chart repo URL" in the Console backend( reads from env ). The UI PR https://github.com/openshift/console/pull/3850 can then consume it after we merge this PR.

2. Make another PR to enhance "Console operator" / CRD to take in the configurable "helm chart repo URL" ( also has a default ) and have the controller set it in Console backend as a config/env.

Reply:
Agree, the default should go in through the `operator`.  and I'd be more inclined to do so via the `config.yaml` than via a flag, but that can be for future discussion.

Reply:
(my mistake, we do support both, config & flags, i forget this little quirk :) )